proto/topodata.proto

```diff
@@ -157,16 +157,14 @@ message Tablet {
   //
   vttime.Time primary_term_start_time = 14;
 
-  // db_server_version represents the database version used by the tablet.
-  string db_server_version = 15;
-
```

Comment:
Is this removal worth a note in the release notes?

Reply:
I don't know? Do we generally add release notes for what are (afaik?) internal fields for Vitess?

Reply:
We don't need this in the release notes.